code/game/machinery/doors/airlock.dm

```diff
@@ -1016,6 +1044,24 @@ About the new airlock wires panel:
 	if(!forced)
 		if( !hasPower() || isWireCut(AIRLOCK_WIRE_OPEN_DOOR) )
 			return 0
+	if(sabotaged && sabotageTurf)
+		p_open = 1
+		update_icon()
+		visible_message("<span class='warning'>[src]'s panel is blown off in a spray of deadly shrapnel!</span>")
+		explosion(sabotageTurf,-1,1,1)

```

Comment:
Bluespace also requested that the doorcharge can be disarmed by opening the panel and cutting it. 

Bluespace also specified "Should an unfortunate open the door from the side the charge was applied, they take critical damage if directly on the tile in front of the door, and almost crit damage directly to the side. Being one tile away from the door causes light damage, and no damage to the side and one tile away. "
But this isn't the case; currently only the person directly on the sabotageTurf gets damage.